Makefile

```diff
@@ -47,3 +47,7 @@ $(CURDIR)/bin/disco
 test:| go_version_check
 	@go test -cover $(PACKAGES)
 
+test-coverage:| go_version_check
```

Comment:
I have added another PR #630 integrating codecov, can you leave this untouched?

Reply:
Removed